catalyst/core/callbacks/wrapper.py

```diff
@@ -29,18 +31,28 @@ def _filter_fn_from_epochs(
 
     if isinstance(epochs, (list, tuple)):
         epochs = sorted(set(epochs))
-        filter_fn = lambda stage, epoch, loader: epoch in epochs
+        if reverse_condition:
+            filter_fn = lambda stage, epoch, loader: epoch not in epochs
+        else:
+            filter_fn = lambda stage, epoch, loader: epoch in epochs
```

Comment:
do you think epoch list would be user friendly?
from my point of view most of the time you want to execute something `one time per N epochs`

maybe we should change the above condition to `epoch % specified_epoch == 0`?

Reply:
@Scitator not everywhere.
i think we should keep both:
- if specified number (for example `N`) then perform some periodical disable/enable every `N` epochs
- if specified list of numbers then perform disable/enable at specified epochs

also, if we touch `epochs` did we expect similar logic from `loaders` argument?
in current implementation it supports only list of epochs.
do we need the same logic for `loaders`:

```python
loaders={
     loader1: number,  # should we do some periodical action for this loader?
     loader2: [number1, number2, number3],  # disable/enable callback for specified epochs
}
```

?

Reply:
🤔  I think, you are right about 2 cases support for epochs
nevertheless, for loaders I suppose we only have enable/disable action